### ISSUE TITLE
Released Feb 26, 2020:  Amazon MSK can now stream broker logs to CloudWatch Logs, Amazon S3, or Amazon Elasticsearch Service

### ISSUE THREAD

**Issue by Sk8rMarc** [terraform-providers/terraform-provider-aws#12550]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* aws_msk_cluster

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.

resource "aws_msk_cluster" "cluster" {
...
   logging_info {
      broker_logs {
         cloud_watch_logs {
            enabled = bool
            log_group = string
         }
         firehose {
            delivery_stream = string
            enabled = bool
         }
         s3 {
            bucket = string
            enabled = bool
            prefix = string
         }
      }
   }
...
}
```
[AWS::MSK::Cluster BrokerLogs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-msk-cluster-brokerlogs.html)

[update-monitoring — AWS CLI 1.18.30 Command Reference](https://docs.aws.amazon.com/cli/latest/reference/kafka/update-monitoring.html)

logging_info (optional) configuration block, update requires no interruption

broker_logs (required) configuration block, update requires no interruption

cloud_watch_logs (optional) configuration block, update requires no interruption
enabled (required) bool, update requires no interruption
log_group (optional) string, update requires no interruption

firehose (optional) configuration block
delivery_stream (optional) string, update requires no interruption
enabled (required) bool, update requires no interruption

s3 (optional) configuration block,  update requires nointerruption
bucket (optional) string, update requires no interruption
enabled (required) bool, update requires no interruption
prefix (optional) string, update requires no interruption

### References

* #0000

**Comment by DrFaust92** [comment]
duplicate of #12186
and may be closed by #12215

**Comment by bflad** [comment]
Support for the `logging_info` configuration block in the `aws_msk_cluster` resource has been merged and will release with version 2.55.0 of the Terraform AWS Provider, later today. Thanks to @arafsheikh for the implementation. 👍
